### DISCUSSION TITLE
Recorded videos as live streaming.

### DISCUSSION THREAD

**Discussion by SauravTirkey** [ant-media/Ant-Media-Server#5339] [Q&A]
Is it possible to stream a pre recorded video as live  with antmedia server? If yes then how? Could you provide the page documentation if there is any?

**Comment by Mohit-3196** [comment]
Hi @SauravTirkey,
How are you?

Yes, you can stream the VoD files/ recorded videos as a live stream with Ant Media Server.
You can easily create a playlist with these files and stream as needed. Please refer to this document for the same https://antmedia.io/docs/guides/publish-live-stream/playlist/#how-to-create-a-playlist

Let me know if there are any more questions/issues.

Cheers,
Mohit

**Comment by SauravTirkey** [comment]
Hi,

Thank you for your reply.

Thanks and regards
Saurav.
